### ISSUE TITLE
INFO: Date formats in database

### ISSUE THREAD

**Issue by georgeef** [moneymanagerex/mmex-ios#116]
## Date fields

| table              | field              | format                |
| ------------------ | ------------------ | --------------------- |
| CURRENCYHISTORY_V1 | CURRDATE           | yyyy-MM-dd            |
| ACCOUNTLIST_V1     | INITIALDATE        | yyyy-MM-dd            |
| ACCOUNTLIST_V1     | STATEMENTDATE      | yyyy-MM-dd            |
| ACCOUNTLIST_V1     | PAYMENTDUEDATE     | yyyy-MM-dd            |
| ASSETS_V1          | STARTDATE          | yyyy-MM-dd            |
| STOCK_V1           | PURCHASEDATE       | yyyy-MM-dd            |
| STOCKHISTORY_V1    | DATE               | yyyy-MM-dd            |
| CHECKINGACCOUNT_V1 | TRANSDATE          | yyyy-MM-dd'T'HH:mm:ss |
| CHECKINGACCOUNT_V1 | LASTUPDATEDTIME    | yyyy-MM-dd'T'HH:mm:ss |
| CHECKINGACCOUNT_V1 | DELETEDTIME        | yyyy-MM-dd'T'HH:mm:ss |
| BILLSDEPOSITS_V1   | TRANSDATE          | yyyy-MM-dd'T'HH:mm:ss |
| BILLSDEPOSITS_V1   | NEXTOCCURRENCEDATE | yyyy-MM-dd            |

## Date values in `INFOTABLE_V1`

| INFOVALUE          | format     |
| ------------------ | ---------- |
| CREATEDATE         | yyyy-MM-dd |
| *_FILTER_LAST_USED | yyyy-MM-dd |

## Note

If a field with format `yyyy-MM-dd'T'HH:mm:ss` has a value of the form `yyyy-MM-dd`, it is padde
d with `T00:00:00`.

**Comment by georgeef** [comment]
@guanlisheng 
Please review the table, then I'll implemented it in `Data`.

**Comment by guanlisheng** [comment]
yup, this is the case as the original mmex4desktop uses wxWidget's `FormatISOCombined` [doc](https://docs.wxwidgets.org/3.2.5/classwx_date_time.html#aab88bde1bbc43049020ac1aa900f5cd8)

```
Returns the combined date-time representation in the ISO 8601 format "YYYY-MM-DDTHH:MM:SS".
```

**Comment by guanlisheng** [comment]
let's close this issue as your impl is perfect now.

**Comment by guanlisheng** [comment]
i c. writing it in wiki would be another way.
